src/components/AttachmentCarousel/CarouselActions/index.js

```diff
@@ -40,6 +40,8 @@ class Carousel extends React.Component {
      * @param {Object} e
      */
     handleKeyPress(e) {
+        // prevents focus from highlighting around the modal
+        e.target.blur();
```

Comment:
I did some testing and noticed that when using the keyboard arrow keys, the focus doesn't change away from the modal which adds the outline when we try to cycle through the images. That differs from using the display arrow icons, because the focus changes back to the body once we click on the screen, which prevent the outline when we cycle through images.

https://user-images.githubusercontent.com/22219519/212750797-3f738239-debb-4555-9831-d18193cd7b4b.mov

TBH I'm not too sure how else to solve the issue.

Reply:
I think this is the only pending issue now. @chiragsalian do you have any suggestions?

Reply:
I looked into this for a while and yeah while that does look like a hack I could not find a better solution. We could add `styles.noOutline` in a couple of places but I think the blur solution might be better.

I _think_ maybe part of the problem is with our ImageView component since i don't see the same problem with our PDFView component. 

I see that PDFView component uses `styles.noSelect` while ImageView doesn't have it. I tried playing around with this a little more but it feels like there are multiple spots to add `styles.noSelect` or `styles.noOutline`. A proper solution would be to check where those styles are needed. 

But I'm fine with moving ahead with blur.

Reply:
Sounds good! Let's move forward with the current solution then!